rmap-loader-deposit-disco/src/main/java/info/rmapproject/loader/deposit/disco/DiscoDepositConsumer.java

```diff
@@ -139,8 +138,7 @@ public void accept(HarvestRecord record) {
             if (response.getStatusLine().getStatusCode() == 201) {
                 harvestRegistry.register(
                         record.getRecordInfo(),
-                        URI.create(response.getFirstHeader("Location")
```

Comment:
This was causing an error because the getUpdateUri method was appending the Location URL to the disco endpoint.  This was resulting in something like this for the update path: https://rmap-hub.org/api/discos/https%3A%2F%2Frmap-hub.org%2Fapi%2F... etc.  I could either just set the location header as the update path, or save the actual DiSCO URI in the database. I chose the latter since there maybe other uses for the value in the URI field.